exp/yaml_migrate/yaml_migrate.go

```diff
@@ -0,0 +1,57 @@
+package yaml_migrate
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"gopkg.in/yaml.v2"
+)
+
+//TODO: Better Error Handling
```

Comment:
Let's go ahead and do the error handling. Rather than panic-ing, returning errors would work better.